### PR TITLE
Added logic to move items back and fourth amongst arrays, also incorp…

### PR DESCRIPTION
**Pull request by CMadison**
**Deliverable Submission**

Please describe your comfort and completeness levels before submitting.

Comfort Level (1-5): 99999999999999999999

Completeness Level (1-5): 9999999999999999999

What did you think of this deliverable?: This is too much work for one hw assignment, especially during a project week. We poor humans do have limits.

…orated seed file. Next steps include working on win functionality via button, marking non correct placements with a red background, and randomizing initial array positions.

### CODE REVIEW

File: js/app.js [hunk 1]
```diff
@@ -1,4 +1,48 @@
...
+    $scope.fruits.splice(idx, 1);
+    console.log(fruits);
+    console.log(vegetables);
+    console.log($scope.fruits.sort);
+
+  // }
+
+  // $scope.checkWin = function(idx) {
+  //   console.log($scope.fruits.sort);
+
```
Comment (scottheron) [inline js/app.js:43]: here is my win check function:

```
$scope.winCheck = function(){

    for (var i = 0; i < $scope.allFruit.length; i++) {
        if ($scope.originalFruit.indexOf($scope.allFruit[i]) == -1) {
            $scope.fail = true;
            return false;
        }
    }
    for (var i = 0; i < $scope.allVegetables.length; i++) {
        if ($scope.originalVegetables.indexOf($scope.allVegetables[i]) == -1) {
            $scope.fail = true;
            return false;
        }
    }
    $scope.win = true;
    console.log($scope.win);
}
```

File: js/app.js [hunk 2]
```diff
@@ -1,4 +1,48 @@
...
+  $scope.moveBackRight = function(idx) {
+    console.log('You have opted to move this item oine div to the right');
+    console.log(idx);
+    $scope.foods.push($scope.fruits[idx]);
+    $scope.fruits.splice(idx, 1);
+    console.log(fruits);
+    console.log(vegetables);
+    console.log($scope.fruits.sort);
+
```
Comment (scottheron) [inline js/app.js]: Here is how I moved from one array to the next (to the fruit column):

    $scope.toFruit = function(idx){
        $scope.allFruit.push($scope.fruitAndVeg[idx]);
        $scope.fruitAndVeg.splice(idx, 1);
        if ($scope.fruitAndVeg.length == 0) {
            $scope.winCheck();
        }
    }

File: index.html [hunk 3]
```diff
@@ -5,22 +5,35 @@
...
         <link rel="stylesheet" href="css/style.css">
 
-        <script src="https://cdnjs.cloudflare.com/ajax/libs/angular.js/1.4.8/angular.min.js"></script>
-        <script src="js/fruits_and_veggies_test.js"></script>
+        <script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.5.7/angular.js"></script>
+        <script src="js/fruits_and_veggies.js"></script>
         <script src="js/app.js"></script>
     </head>
-    <body>
+    <body ng-app="F&V" ng-controller="mainCtrl">
```
Comment (scottheron) [inline index.html]: Here is my HTML for the fruits:

    <div class="col-sm-4">
        Fruits
        <div ng-repeat="aFruit in allFruit track by $index" ng-class="{'has-fail': !win, 'has-fail': fail}" class="well">
            {{ aFruit }}
            <button ng-click="toPoolFromFruit($index)" type="button" class="btn btn-primary"><span class="glyphicon glyphicon-arrow-right"></span></button>
        </div>
    </div>

File: index.html [hunk 4]
```diff
@@ -5,22 +5,35 @@
...
                 <div class="col-sm-4">
-                    col 3
+                    Veggies
+                    <div ng-repeat="veggie in veggies track by $index" class="well">
+                    <button ng-click="moveBackLeft($index)" class="btn btn-primary">< </button>
+                    {{veggie}}
+                    </div>
                 </div>
             </div>
         </div>
```
Comment (scottheron) [inline index.html]: here is my win check function:
    
    $scope.winCheck = function(){
        
        for (var i = 0; i < $scope.allFruit.length; i++) {
            if ($scope.originalFruit.indexOf($scope.allFruit[i]) == -1) {
                $scope.fail = true;
                return false;
            }
        }
        for (var i = 0; i < $scope.allVegetables.length; i++) {
            if ($scope.originalVegetables.indexOf($scope.allVegetables[i]) == -1) {
                $scope.fail = true;
                return false;
            }
        }
        $scope.win = true;
        console.log($scope.win);
    }

File: index.html (hunk 3 above)
Comment (scottheron) [inline index.html:12]: Here is my banner for winning:

```
<div ng-show="win" ng-class="{'has-success': win}">
    Congrats! You Won!
</div>
```

File: index.html [hunk 5]
```diff
@@ -5,22 +5,35 @@
...
 
-        <script src="https://cdnjs.cloudflare.com/ajax/libs/angular.js/1.4.8/angular.min.js"></script>
-        <script src="js/fruits_and_veggies_test.js"></script>
+        <script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.5.7/angular.js"></script>
+        <script src="js/fruits_and_veggies.js"></script>
         <script src="js/app.js"></script>
     </head>
-    <body>
+    <body ng-app="F&V" ng-controller="mainCtrl">
         <div class="container">
```
Comment (scottheron) [inline index.html:13]: Here is my CSS for red and green on win or fail:

```
.has-success {
    background-color: green;
}

.has-fail {
    background-color: red;
}
```

File: index.html [hunk 6]
```diff
@@ -5,22 +5,35 @@
...
             <h1>Fruit and Veggies</h1>
```
Comment (scottheron) [inline index.html:14]: Here is my HTML for the fruits:

```
<div class="col-sm-4">
    Fruits
    <div ng-repeat="aFruit in allFruit track by $index" ng-class="{'has-fail': !win, 'has-fail': fail}" class="well">
        {{ aFruit }}
        <button ng-click="toPoolFromFruit($index)" type="button" class="btn btn-primary"><span class="glyphicon glyphicon-arrow-right"></span></button>
    </div>
</div>
```

**Comment by scottheron** [comment]
good start
